docs/content/how-to/kubevirt/configuring-network.md

```diff
@@ -32,3 +32,29 @@ hcp create cluster kubevirt \
 In this example, the KubeVirt VMs will have interfaces attached to the networks
 for the NetworkAttachmentDefinitions network1 and network2 which reside in
 namespace my-namespace.
+
+## Using managed networks as default
+
+Users managing a network (DHCP, routing, etc...) can use that network 
+as the default one for the kubevirt hosted clusters, to do so
+disable pod default network and attach an additional one that connects to it 
+with the hcp command line tool arguments `--attach-default-network=false` and 
+`--additional-network`.
```

Comment:
what happens if someone sets `-attach-default-network=false` then adds multiple addtional networks? Which one will be the default?

Reply:
If have just remeber that we activated nodeip-configuration at MCO (https://github.com/openshift/machine-config-operator/pull/4039), with that we can do what is stated at https://docs.openshift.com/container-platform/4.15/support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues